integration-tests/devtools/src/test/java/io/quarkus/platform/catalog/RegistrySnapshotCatalogCompatibilityTest.java

```diff
@@ -25,6 +25,7 @@
 import io.quarkus.registry.catalog.PlatformRelease;
 import io.quarkus.registry.catalog.PlatformStream;
 
+@Disabled
 public class RegistrySnapshotCatalogCompatibilityTest {
 
     private final ExtensionCatalogResolver catalogResolver = QuarkusProjectHelper.getCatalogResolver();
```

Comment:
if the issue is about the init happening when we don't want it to is it not better to have this setup in a @Before ? somethinglike:

```suggestion
    private ExtensionCatalogResolver catalogResolver;
    
    @Before
    public void init() {
    catalogResolver = QuarkusProjectHelper.getCatalogResolver();
    }
```

Reply:
Yes, that should also work